src/installation/musl.md

```diff
@@ -48,9 +48,9 @@ Use [chroot(1)](https://man.voidlinux.org/chroot.1) to change to the new root,
 then run glibc programs as usual. Once you've finished using it, unmount the
 chroot using [umount(8)](https://man.voidlinux.org/umount.8).
 
-#### PRoot
+#### bubblewrap
```

Comment:
```suggestion
#### Bubblewrap
```

Reply:
From the `bubblewrap` [README](https://github.com/containers/bubblewrap/blob/master/README.md) it was not clear to me whether the official name is stylized with capital or lower "B" and as neither `musl`, nor `glibc chroot` section titles are capitalized, I chose the lowercase variant which seemed more likely based on the README. However, the README uses capital "B" in the title so indeed let's use that.